workflows/autometa_flagged.sh

```diff
@@ -0,0 +1,432 @@
+#!/usr/bin/env bash
+#SBATCH --nodes=1
+#SBATCH --ntasks-per-node=16
+#SBATCH --partition=queue
+#SBATCH --time=14-00:00:00
+#SBATCH --error=autometa.%J.err
+#SBATCH --output=autometa.%J.out
+
+usage()
+{
+   # Display Help
+   echo "-h     Display help menu"
+   echo
+   echo "Required flags:"
+   echo "-o     Directory path to where the output will go"
+   echo "-a     Full path to where your scaffold/contig file is (include file name)"
+   echo "-s     Simple name to prefix output files"
+   echo "-n     Path to NCBI databases directory"
+   echo "-m     Path to marker files directory"
+   echo "-l     Contig length cutoff in base pairs. (NOTE: Change this according to the metagenome assembly's N50)"
```

Comment:
If you are planning on updating the quickstart using this template, it may be worth showcasing `seqkit` as this is easy to install via conda (or mamba) and can quickly calculate various metagenome statistics with a simple command.

```bash
seqkit stat --all --tabular <metagenome(s)>
```

Reply:
Sweet - can do :)